<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_teacher_dialog</class>
 <widget class="QDialog" name="Edit_teacher_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>93</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>specialization</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLineEdit" name="ID_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="email_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="specialization"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>189</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="day_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="month_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="year_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="last_name_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="first_name_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phone_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Birth:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>last_name_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>first_name_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phone_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ID_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>email_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specialization</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>day_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>month_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>year_2</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Save</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
